Author: Andreas Tille <tille@debian.org>
Last-Update: 2025-06-13
Bug-Debian: https://bugs.debian.org/1097851
Description: Modern prototype syntax to build with gcc-15

--- a/atomicio.c
+++ b/atomicio.c
@@ -34,11 +34,7 @@
  * ensure all of data on socket comes through. f==read || f==write
  */
 ssize_t
-atomicio(f, fd, _s, n)
-	ssize_t (*f) ();
-	int fd;
-	void *_s;
-	size_t n;
+atomicio(ssize_t (*f)(int, void *, size_t), int fd, void *_s, size_t n)
 {
 	char *s = _s;
 	ssize_t res, pos = 0;
